SwiftReflector/Importing/TypeAggregator.iOS.cs

```diff
@@ -477,6 +487,11 @@ public partial class TypeAggregator {
 			// CoreImage
 			{ "CoreImage.CIDataMatrixCodeEccVersion", "CIDataMatrixCodeDescriptor.ECCVersion" },
 			{ "CoreImage.CIQRCodeErrorCorrectionLevel", "CIQRCodeDescriptor.ErrorCorrectionLevel" },
+			// CoreMedia
+			// iOS 13.0 or later: { "CoreMedia.CMBlockBufferError", "CMBlockBuffer.Error" },
+			// iOS 13.0 or later: { "CoreMedia.CMClockError", "CMClock.Error" },
+			// iOS 13.0 or later: { "CoreMedia.CMSampleBufferError", "CMSampleBuffer.Error" },
+			// iOS 13.0 or later: { "CoreMedia.CMSyncError", "CMSync.Error" },
```

Comment:
@stephen-hawley is this the correct approach?

Reply:
To add context (not sure if this is correct or not either): these were the replacements suggested by Swift's errors, but when I tried to compile with these replacements it threw the version requirement. So I left the replacements in as comments for the time being (to make it easier for if someone goes through and updates these for iOS 13) and put them in namespaces to ignore.

Reply:
👍 We're not compiling to 13.0 (yet)